llm-training-course/colab/finetuning.ipynb

```diff
@@ -0,0 +1,1100 @@
+{
```

Comment:
<p>we don't have to pass <code>report_to = "wandb"</code> if using the callback? Maybe show it anyways to get people in the habit of it anyways when using HF Trainer.</p><p> </p><p>Link to W&amp;B HF documentation above this cell too</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/edu/pull/135/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>updated</p>